<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gestion</class>
 <widget class="QDialog" name="Gestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>616</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gestion Abonne</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#QWidget {

background:#404040;

}

QTabWidget::pane { /* The tab widget frame */

border-top: 5px solid #535353;

}

QTabWidget::tab-bar {

left: 0px; /* move to the right by 5px */

}

/* Style the tab using the tab sub-control. Note that it reads QTabBar _not_ QTabWidget */

QTabBar::tab {

background: #dcdcdc;

border: 2px solid #535353;

border-bottom-color: rgb(0, 0, 0); /* same as the pane color */

border-top-left-radius: 7px;

border-top-right-radius: 7px;

min-width: 120px;

padding: 8px;

color: rgb(0, 0, 184);

font-size:15px;

font-family:Arial, Helvetica, sans-serif;

}

QTabBar::tab:selected, QTabBar::tab:hover {

background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #c3a1a1, stop: 0.4 rgb(85, 0, 255), stop: 0.5 rgb(67, 0, 202), stop: 1.0 #3f00bd);
color :white;
}

QTabBar::tab:selected {

border-color: rgb(0, 0, 0);

border-bottom-color: #535353; /* same as pane color */

color: #dcdcdc;

font-size:15px;

font-family:Arial, Helvetica, sans-serif;

}

QTabBar::tab:!selected {

margin-top: 2px; /* make non-selected tabs look smaller */

}

</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>701</width>
     <height>491</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>701</width>
     <height>491</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="Preference">
    <attribute name="title">
     <string>Preference</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Critère de recherche</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_preference">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="currentText">
      <string>Cliquer pour choisir le critére</string>
     </property>
     <item>
      <property name="text">
       <string>Cliquer pour choisir le critére</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Adresse</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Age</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sexe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Niveau d'etude</string>
      </property>
     </item>
    </widget>
    <widget class="QTableView" name="tab_preference">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>681</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Preference">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Classification">
    <attribute name="title">
     <string>Classification</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Classer Selon : </string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Nom">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ordre Alphabitique des noms</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_AGEASC">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ordre d'age croissant  </string>
     </property>
    </widget>
    <widget class="QTableView" name="tab_classification">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>621</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_AGEDESC">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ordre d'age decroissant  </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_ETUDEASC">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ordre du niveau d'etude croissant</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_ETUDEDESC">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>90</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Ordre du niveau d'etude decroissant</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Reclamation">
    <attribute name="title">
     <string>Reclamation</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_EnvoyerReclamation">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>420</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Envoyer </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_AnnulerRec">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>420</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Ecrire la reclamation ci-desous</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_reclamation">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>601</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ID_C">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Identifiant du centre :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Statstiques</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_changePoste">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Changer Poste</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_changeCompte">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>550</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Changer Compte</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_IMPRIME">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>10</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>IMPRIMER</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
